static_src/components/breadcrumbs/breadcrumbs.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { appPropType } from '../../stores/app_store';
+import { orgPropType } from '../../stores/org_store';
+import { spacePropType } from '../../stores/space_store';
+import Item from './breadcrumbs_item';
```

Comment:
Let's keep the name the same as what's in the imported file.
`Item` -> `BreadcrumbsItem`

Reply:
@jcscottiii I was just trying to remove some stutter but I do concede that keeping `BreadcrumbsItem` makes searching easier (just at the expense of stutter) - but all good, I will update.